app/serializers/api/variant_serializer.rb

```diff
@@ -41,10 +41,10 @@ def thumb_url
   end
 
   def unit_price_price
-    (rand * 10).round(2)
+    price_with_fees / VariantUnits::UnitPrices.new(object).unit_price_denominator
```

Comment:
Minor performance note: can we memoize `VariantUnits::UnitPrices.new(object)` in a private method? I think it'll be more efficient and maybe avoid some superfluous database hits.

Reply:
Like this? 651e11230eb67028bd677a25164aa7af8fc1d54b

Reply:
Thanks :)